staticcheck.conf

```diff
@@ -0,0 +1 @@
+dot_import_whitelist=["github.com/onsi/gomega", "github.com/onsi/gomega/gstruct"]
```

Comment:
Please add an empty line at the end of the file

Reply:
I agree with adding `github.com/onsi/gomega` but do we really need to ignore `gstruct`?

It seems `gstruct` is imported only twice, therefore I think we should change these in a proper way.